pkg/manifest.go

```diff
@@ -107,5 +108,8 @@ func ReadInManifest(manifestFile string) (model.InputManifest, error) {
 	if err := validateManifestDependencies(manifest.Dependencies); err != nil {
 		return manifest, fmt.Errorf("invalid manifest: %v", err)
 	}
+	if _, err = semver.StrictNewVersion(manifest.Version); err != nil {
```

Comment:
I think this will fail for 1.5-dev.sha

Reply:
can we name the release 1.5.0-dev.sha? So that all of our versions in the future will be semver.